Infineon/cmsis-packs
# Add Cypress.CY8CPROTO-062-4343W_BSP.0.1.0.pack

Testing:

1. Check Cypress.CY8CPROTO-062-4343W_BSP.pdsc schema against https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/Utilities/PACK.xsd - PASS

2. Check the board is visible in Keil Pack Installer - PASS

![image](https://user-images.githubusercontent.com/45975200/137163457-90528969-c508-42da-a39b-9d0213d8074b.png)


3. PackChk.exe CY8CPROTO-062-4343W_BSP/Cypress.CY8CPROTO-062-4343W_BSP.pdsc - PASS (2 non-fatal warnings):

```
PackChk.exe 1.3.87 (Jul  2 2019 08:31:52)
Copyright (C) 2012-2019 ARM Ltd and ARM Germany GmbH. All rights reserved.

Phase1: Read PDSC files

Phase2: Static Data & Dependencies check

*** WARNING M379: C:\git\github\Infineon\cmsis-packs\CY8CPROTO-062-4343W_BSP\Cypress.CY8CPROTO-062-4343W_BSP.pdsc (Line 36)
  No example(s) found for Board '[Cypress] CY8CPROTO-062-4343W'.

*** WARNING M382: C:\git\github\Infineon\cmsis-packs\CY8CPROTO-062-4343W_BSP\Cypress.CY8CPROTO-062-4343W_BSP.pdsc (Line 25)
  Requirement <package> '[Cypress] PSoC6_DFP 1.2.0' could not be resolved.

Phase3: RTE Model based Data & Dependencies check

Found 0 Error(s) and 2 Warning(s).
```

I was not able to test if any of the <book> links are correctly parsed by Keil uVision IDE.
